ui/components/component-library/modal-content/modal-content.scss

```diff
@@ -6,6 +6,11 @@
   overflow: auto;
   overscroll-behavior-y: none;
 
+  // Maximize dialog visibility on small screen heights
+  @media (max-height: 475px) {
+    padding: 8px;
+  }
+
```

Comment:
Adding edgecase styles for smaller screen heights. Need to decide on a height value here that makes sense

### Before
`ModalContent` without  these styles

https://github.com/MetaMask/metamask-extension/assets/8112138/e7cbca57-27db-48af-ae7f-5b0e1630ed71


### After
`ModalContent` with these styles maximizes content view-ability 

https://github.com/MetaMask/metamask-extension/assets/8112138/e5631141-7e92-410b-848b-f4447f6d5a31